etc/zones/db.asuc.org

```diff
@@ -80,4 +80,7 @@ www.dev	IN CNAME	death.OCF.Berkeley.EDU.
 cfo	IN CNAME	death.OCF.Berkeley.EDU.
 www.cfo	IN CNAME	death.OCF.Berkeley.EDU.
 
+octo	IN CNAME	unruffled-shockley-567ba0.netlify.com
+www.octo	IN CNAME	unruffled-shockley-567ba0.netlify.com
```

Comment:
These need a period at the end of the line so that bind knows these aren't relative CNAMES (otherwise it'll try and make these `unruffled-shockley-567ba0.netlify.com.asuc.org`)

Reply:
ah, ye. ty